### ISSUE TITLE
Working with Rademacher Start2Smart Bridge?

### ISSUE THREAD

**Issue by Krillle** [jortgies/homebridge-rademacher-blinds#15]
Is this plugin working with Rademacher Start2Smart Bridge [1] as well, or is HomePilot needed?

[1] https://www.rademacher.de/smart-home/produkte/start2smart

**Comment by jortgies** [comment]
If the start2smart bridge also has an API accessible on http://<bridge-ip>/v4/devices you can probably use this plugin.

**Comment by Krillle** [comment]
The bridge seems to be the smaller sister, serving Alexa and such. So I assume it is offering the same API. 
I will check out and report.

**Comment by SayWhat69** [comment]
> The bridge seems to be the smaller sister, serving Alexa and such. So I assume it is offering the same API.
> I will check out and report.

any result yet?

**Comment by Krillle** [comment]
Yes, Homebridge Rademacher Homepilot plugin [1] works flawless with the Rademacher Start2Smart Bridge.

[1] https://github.com/bombadiltom/homebridge-rademacher-homepilot#readme

**Comment by SayWhat69** [comment]
Great news! Thanks a lot!

**Comment by amddeus** [comment]
@Krillle Does the Start2Smart Bridge work when you block internet and only use it locally via homebridge/kit?

**Comment by Krillle** [comment]
Yes, it should. It's a local system.
